### DISCUSSION TITLE
gpt-4-vision-preview model support

### DISCUSSION THREAD

**Discussion by Alpha162** [n3d1117/chatgpt-telegram-bot#451] [General]
I have noticed 'gpt-4-vision-preview' access has been allowed:

https://platform.openai.com/docs/guides/vision

Looks like it would require some work to support, but having GPT4-V ability direct from Telegram would be pretty amazing.

**Comment by andy68man** [comment]
Completely agree, being able to quickly share images via telegram would be amazing!

**Comment by chxmontoro** [comment]
Bumbing on this. Any luck with supporting gpt-4-vision-preview yet?
